static/templates/message_body.hbs

```diff
@@ -50,7 +50,7 @@
     <div class="message_edit_form"></div>
 </div>
 <div class="message_expander message_length_controller" title="{{t 'Expand message (-)' }}">{{t "[More…]" }}</div>
-<div class="message_condenser message_length_controller" title="{{t 'Condense message (-)' }}">{{t "[Condense message]" }}</div>
+<div class="message_condenser message_length_controller" title="{{t 'Show less (-)' }}">{{t "[Show less]" }}</div>
```

Comment:
Should we pull the `[]` and ` (-)` out of the translation syntax?  That'd result in there being only one copy of this string.

Also, do we want to use "Show more" for analogy?

Reply:
Pushed a revision that takes the `[]` and `(-)` out of the translation tag.

I'm personally inclined to value brevity more than consistency when it comes to `[More…]` — unlike "Condense message", I don't see "More" being confusing.